fixtures/beamz/cob30rgb-flatpar.json

```
@@ -0,0 +1,277 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "COB30RGB FlatPAR",
+  "shortName": "COB30RGB",
+  "categories": ["Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Pyronaut"],
+    "createDate": "2023-04-08",
+    "lastModifyDate": "2023-04-08"
+  },
+  "links": {
+    "manual": [
+      "https://sonomateriel.com/amfile/file/download/file/931/product/6491/"
+    ],
+    "productPage": [
+      "https://www.beamzlighting.com/product/cob30rgb-flatpar/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=85frQfcbPpw"
+    ]
+  },
+  "physical": {
+    "dimensions": [170, 220, 200],
+    "weight": 1.1,
+    "power": 40,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
```

```suggestion
      "capabilities": [
        {
          "dmxRange": [0, 9],
          "type": "NoFunction"
        },
        {
          "dmxRange": [10, 255],
          "type": "ShutterStrobe",
          "shutterEffect": "Strobe",
          "speedStart": "slow",
          "speedEnd": "fast"
        }
      ]
```

One strobe channel says 0-100% at 0-255, while the other says 0-100% at 10-255.  I think it's likely they both have the latter functionality.